src/azure/cli/commands/arm.py

```diff
@@ -228,8 +231,7 @@ def one_required(namespace):
                      metavar='LIST KEY=VALUE')
     cmd.add_argument('properties_to_remove', '--remove', nargs='+', action=OrderedArgsAction,
                      default=[], help='Remove a property or an element from a list.  Example: '
-                     '--remove property.list <index>', metavar='LIST INDEX',
-                     validator=one_required)
+                     '--remove property.list <index>', metavar='LIST INDEX')
```

Comment:
>LIST INDEX [](start = 64, length = 10)

I believe metavar is supposed to be one word or use _ instead of space to avoid confusing the user who might interpret it as needing two values.